src/modules/SX127x/SX1278.cpp

```diff
@@ -315,8 +315,10 @@ int16_t SX1278::setOutputPower(int8_t power) {
 }
 
 int16_t SX1278::setGain(uint8_t gain) {
+    int16_t modem = getActiveModem();
+
   // check active modem
-  if(getActiveModem() != SX127X_LORA) {
+  if(modem != SX127X_LORA && modem != SX127X_FSK_OOK) {
```

Comment:
This check is redundant now, the modem type is determined by a single bit, so there's only two options.

Reply:
perfect, removed!